tests/e2e/actor_sdks/actor_sdks_test.go

```diff
@@ -172,8 +172,8 @@ func TestActorInvocationCrossSDKs(t *testing.T) {
 		healthCheckApp(t, externalURL, numHealthChecks)
 	}
 
-	t.Log("Sleeping for 10 seconds ...")
-	time.Sleep(10 * time.Second)
+	t.Log("Sleeping for 15 seconds ...")
```

Comment:
I am afraid that sleep is not a good solution :(

Reply:
Just my guess, to solve part of the e2etest doesn't pass. But it seems not working. I will roll back this change.